apps/web/src/pages/TalentNominations/NominationGroup/components/NominatedForList.tsx

```diff
@@ -0,0 +1,91 @@
+import { useIntl } from "react-intl";
+
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+
+import ComputedIcon from "./ComputedIcon";
+
+export const NominationGroupSidebarForList_Fragment = graphql(/* GraphQL */ `
+  fragment NominationGroupSidebarForList on TalentNominationGroup {
+    advancementNominationCount
+    advancementDecision {
+      value
+    }
+    lateralMovementNominationCount
+    lateralMovementDecision {
+      value
+    }
+    developmentProgramsNominationCount
+    developmentProgramsDecision {
+      value
+    }
+  }
+`);
+
+interface NominatedForListProps {
+  nominationGroupSidebarForListQuery: FragmentType<
+    typeof NominationGroupSidebarForList_Fragment
+  >;
+}
+
+const NominatedForList = ({
+  nominationGroupSidebarForListQuery,
+}: NominatedForListProps) => {
+  const intl = useIntl();
+
+  const nominationGroupSidebarForList = getFragment(
+    NominationGroupSidebarForList_Fragment,
+    nominationGroupSidebarForListQuery,
+  );
+
+  const {
+    advancementNominationCount,
+    advancementDecision,
+    lateralMovementNominationCount,
+    lateralMovementDecision,
+    developmentProgramsNominationCount,
+    developmentProgramsDecision,
+  } = nominationGroupSidebarForList;
+
+  return (
+    <ul data-h2-padding="base(0)" data-h2-list-style="base(none)">
+      <li>
+        <ComputedIcon
+          count={advancementNominationCount ?? 0}
+          decision={advancementDecision?.value}
+        />
+        {intl.formatMessage(commonMessages.advancement)}
+        {!!advancementNominationCount && advancementNominationCount !== 0
+          ? // eslint-disable-next-line formatjs/no-literal-string-in-jsx
+            ` (${advancementNominationCount})`
+          : null}
```

Comment:
The number in parentheses seems conditionally rendered, so I think I have to split it out like this

Reply:
I would use plural syntax here.

```
{advacementNominationCount, plural,
   zero {Advacement}
   other {Advacement ({advacementNominationCount, number})}
}
```

Reply:
Oh thanks, that helps a lot
https://github.com/GCTC-NTGC/gc-digital-talent/pull/13100/commits/f4eb8c7f13f29271e2340d9f314ca260fbe9f4ed